edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -265,6 +265,19 @@ export default {
       return this.value.spec.rkeConfig;
     },
 
+    machineSelectorLength() {
+      return this.rkeConfig.machineSelectorConfig.length > 1 ? 'any' : 'all';
+    },
+
+    advancedTitleAlt() {
+      const machineSelectorLength = this.machineSelectorLength;
+
+      return this.t(
+        'cluster.advanced.argInfo.machineSelector.titleAlt',
+        { selection: machineSelectorLength }
```

Comment:
to avoid the untranslated any/all you could pass in the number directly and use that with the `plural` feature. Either with the `>1` or if that doesn't work `=0` and `=1` with same values.

For a good overall example see `promptRemove.andOthers`, or simpler `generic.resource`

Reply:
FYI - using `>1` throws:

```
Expected "=", "offset:", or argName but ">" found. 
```

I opted for the `=1` approach